<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>208</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionMark"/>
   </widget>
   <widget class="QMenu" name="menu_Preview">
    <property name="title">
     <string>&amp;Preview</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionActualSize"/>
    <addaction name="actionFitSize"/>
   </widget>
   <widget class="QMenu" name="menu_Sharpening">
    <property name="title">
     <string>&amp;Sharpening</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Smoothness">
    <property name="title">
     <string>S&amp;moothness</string>
    </property>
    <addaction name="actionGauss"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Preview"/>
   <addaction name="menu_Sharpening"/>
   <addaction name="menu_Smoothness"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionActualSize"/>
   <addaction name="actionFitSize"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionBack">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionActualSize">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/actualsize.png</normaloff>:/images/actualsize.png</iconset>
   </property>
   <property name="text">
    <string>ActualSize</string>
   </property>
   <property name="toolTip">
    <string>ActualSize</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFitSize">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/fitsize.png</normaloff>:/images/fitsize.png</iconset>
   </property>
   <property name="text">
    <string>FitSize</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/rename.jpg</normaloff>:/images/rename.jpg</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionMark">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/mark.png</normaloff>:/images/mark.png</iconset>
   </property>
   <property name="text">
    <string>Mark</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionGauss">
   <property name="text">
    <string>Gauss</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
